tools/bazel.rc

```diff
@@ -80,16 +80,22 @@ build:tsan --test_env=TSAN_OPTIONS=report_atomic_races=0
 build:tsan --action_env=TSAN_OPTIONS=suppressions=test/core/util/tsan_suppressions.txt:halt_on_error=1:second_deadlock_stack=1
 
 build:ubsan --strip=never
+build:ubsan --copt=-fsanitize=undefined
 build:ubsan --copt=-fsanitize-link-c++-runtime
 build:ubsan --copt=-fno-omit-frame-pointer
 build:ubsan --copt=-DGRPC_UBSAN
 build:ubsan --copt=-DNDEBUG
 build:ubsan --copt=-fno-sanitize=function,vptr
+build:ubsan --linkopt=-fsanitize=undefined
 build:ubsan --linkopt=-fsanitize-link-c++-runtime
 build:ubsan --action_env=UBSAN_OPTIONS=halt_on_error=1:print_stacktrace=1:suppressions=test/core/util/ubsan_suppressions.txt
 # For some reasons, these two stopped being propagated, so, redeclaring them here.
 # That's a hack that needs to be removed once we understand what's going on.
 build:ubsan --copt=-DGRPC_PORT_ISOLATED_RUNTIME=1
+# This is required to workaround the known problem, https://bugs.llvm.org/show_bug.cgi?id=16404.
+build:ubsan --linkopt=-lgcc_s
```

Comment:
I tried more or less the same thing yesterday and it didn't work.
(actually it seems it works for running ubsan locally, but not on RBE). We'll have to investigate a bit more.